<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>787</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_main">
    <item>
     <widget class="QListWidget" name="listWidget">
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>DATABASE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CRAWLER</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ANALYSIS</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>BOARD</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>WEB</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>USER</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SETTING</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_database">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableWidget" name="database_tablewidget"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_search">
          <item>
           <widget class="QLineEdit" name="database_searchDB_lineinput">
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="database_searchDB_button">
            <property name="text">
             <string>검색</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="database_chatgpt_button">
            <property name="text">
             <string>ChatGPT</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0">
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <item>
           <widget class="QPushButton" name="database_deleteDB_button">
            <property name="text">
             <string>DB 삭제</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="database_viewDB_button">
            <property name="text">
             <string>DB 조회</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="database_saveDB_button">
            <property name="text">
             <string>CSV로 저장</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_crawler">
       <property name="windowTitle">
        <string>CRAWLER</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QWidget" name="crawler_webview" native="true"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="crawler_server_button">
            <property name="text">
             <string>크롤러 서버</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_data_process">
       <property name="windowTitle">
        <string>ANALYSIS</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_data_process">
        <item>
         <widget class="QTabWidget" name="tabWidget_data_process">
          <property name="tabPosition">
           <enum>QTabWidget::North</enum>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_file">
           <attribute name="title">
            <string>파일 불러오기</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_tab_1">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <layout class="QHBoxLayout" name="analysis_filefinder_layout"/>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="QPushButton" name="analysis_timesplitfile_btn">
                  <property name="text">
                   <string>분할(년/월)</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="analysis_mergefile_btn">
                  <property name="text">
                   <string>CSV 병합</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="analysis_dataanalysisfile_btn">
                  <property name="text">
                   <string>통계 분석</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="analysis_wordcloud_btn">
                  <property name="text">
                   <string>워드클라우드</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="analysis_tokenization_btn">
                  <property name="text">
                   <string>토큰화</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="analysis_kemkim_btn">
                  <property name="text">
                   <string>KEM KIM</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_board">
       <layout class="QVBoxLayout" name="verticalLayout_data_process">
        <item>
         <widget class="QTabWidget" name="tabWidget_board">
          <property name="tabPosition">
           <enum>QTabWidget::North</enum>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_version">
           <attribute name="title">
            <string>패치 노트</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_tab_1">
            <item>
             <widget class="QTableWidget" name="board_version_tableWidget"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QPushButton" name="board_addversion_button">
                <property name="text">
                 <string>추가</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="board_deleteversion_button">
                <property name="text">
                 <string>삭제</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="board_detailversion_button">
                <property name="text">
                 <string>자세히</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_bugreport">
           <attribute name="title">
            <string>버그 리포트</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_tab_1">
            <item>
             <widget class="QTableWidget" name="board_bug_tableWidget"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QPushButton" name="board_addbug_button">
                <property name="text">
                 <string>추가</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="board_deletebug_button">
                <property name="text">
                 <string>삭제</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="board_detailbug_button">
                <property name="text">
                 <string>자세히</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_post">
           <attribute name="title">
            <string>자유게시판</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_tab_1">
            <item>
             <widget class="QTableWidget" name="board_post_tableWidget"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QPushButton" name="board_addpost_button">
                <property name="text">
                 <string>추가</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="board_deletepost_button">
                <property name="text">
                 <string>삭제</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="board_detailpost_button">
                <property name="text">
                 <string>자세히</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="board_editpost_button">
                <property name="text">
                 <string>수정</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_web">
        <layout class="QVBoxLayout" name="verticalLayout">
        <!-- 사이트 바로가기 버튼들 -->
        <item>
        <widget class="QGroupBox" name="group_sites">
            <property name="title">
            <string>Websites</string>
            </property>
            <layout class="QGridLayout" name="grid_sites">
            <item row="0" column="0">
            <widget class="QPushButton" name="web_homepage_button">
            <property name="text"><string>홈페이지</string></property>
            </widget>
            </item>
            <item row="0" column="1">
            <widget class="QPushButton" name="web_sue_button">
            <property name="text"><string>AI 고소장 생성기</string></property>
            </widget>
            </item>
            <item row="0" column="2">
            <widget class="QPushButton" name="web_carnumber_button">
            <property name="text"><string>AI 번호판 인식</string></property>
            </widget>
            </item>
            <item row="0" column="3">
            <widget class="QPushButton" name="web_github_button">
            <property name="text"><string>개발자 GitHub</string></property>
            </widget>
            </item>
            </layout>
        </widget>
        </item>

        <!-- 다운로드 버튼들 -->
        <item>
        <widget class="QGroupBox" name="group_downloads">
            <property name="title"><string>Formats</string></property>
            <layout class="QGridLayout" name="grid_downloads">
            <item row="0" column="0">
            <widget class="QPushButton" name="exception_list_download_btn">
            <property name="text"><string>예외어 사전 양식 다운로드</string></property>
            </widget>
            </item>
            <item row="0" column="1">
            <widget class="QPushButton" name="koreng_list_download_btn">
            <property name="text"><string>KEMKIM 한영변환 양식 다운로드</string></property>
            </widget>
            </item>
            <item row="0" column="1">
            <widget class="QPushButton" name="include_list_download_btn">
            <property name="text"><string>토큰 포함단어 양식 다운로드</string></property>
            </widget>
            </item>
            </layout>
        </widget>
        </item>
        
        <!-- 홈페이지 수정용 레이아웃 -->
        <item>
        <widget class="QGroupBox" name="group_downloads">
            <property name="title"><string>Homepage Info</string></property>
            <layout class="QGridLayout" name="grid_downloads">
            <item row="0" column="0">
            <widget class="QPushButton" name="web_download_doc1_button">
            <property name="text"><string>랩 구성원 추가</string></property>
            </widget>
            </item>
            <item row="0" column="1">
            <widget class="QPushButton" name="web_download_doc2_button">
            <property name="text"><string>논문 추가</string></property>
            </widget>
            </item>
            </layout>
        </widget>
        </item>
        </layout>
      </widget>
      <widget class="QWidget" name="page_user">
       <layout class="QVBoxLayout" name="verticalLayout_user">
        <item>
         <widget class="QTabWidget" name="tabWidget_user">
          <property name="tabPosition">
           <enum>QTabWidget::North</enum>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_userlist">
           <attribute name="title">
            <string>User List</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QTableWidget" name="user_tablewidget"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_input">
              <item>
               <widget class="QLabel" name="label_name">
                <property name="text">
                 <string>이름:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="userName_lineinput"/>
              </item>
              <item>
               <widget class="QLabel" name="label_email">
                <property name="text">
                 <string>이메일:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="user_email_lineinput"/>
              </item>
              <item>
               <widget class="QLabel" name="label_key">
                <property name="text">
                 <string>Key:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="user_key_lineinput"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QPushButton" name="user_deleteuser_button">
                <property name="text">
                 <string>사용자 삭제</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="user_adduser_button">
                <property name="text">
                 <string>사용자 추가</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_info">
       <layout class="QVBoxLayout" name="verticalLayout"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1069</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
